packages/webapp/src/pages/Connection/Show.tsx

```diff
@@ -151,7 +141,7 @@ We could not retrieve and/or refresh your access token due to the following erro
         return <PageNotFound />;
     }
 
-    if (!loaded || !syncLoaded) {
+    if (!loaded || syncLoading) {
```

Comment:
wha's the difference between `!loaded` and `syncLoading`

Reply:
Loaded is the page frame and meta information like the connection and provider and syncLoading is the actual sync data